releasenotes/notes/apm-fix-sql-normalization-of-negative-integers-4ffd29fc5dbcdff3.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Fix bug in SQL normalization that resulted in negative integer values to be normalized with an extra minus sign token.
```

```suggestion
    APM: Fix bug in SQL normalization that resulted in negative integer values to be normalized with an extra minus sign token.
```
We need to do this for all `trace-agent` changes.